server/@types/hmpo-form-wizard/index.d.ts

```diff
@@ -54,7 +54,7 @@ declare module 'hmpo-form-wizard' {
             | 'all'
             | 'some'
           value: Value | MultiValue
-          next: string
+          next: string | null
```

Comment:
how can a next step condition point nowhere?

Reply:
oh you mean when this is the final step? i feel that we should come back to this… firstly, the condition should probably be excluded? or perhaps it should point at the report's overview page (once that is wired up)?

Reply:
It is very similar to how in the config the next question id is null as it's the final question.

> in any case, it feels weird to state a condition that is checked and matches, but ends up being ignored
why tho? If you answer in a certain way it means you're done it was the final question. It doesn't have to be special from the steps perspective (although of course it would be handled differently)

(bearing in mind potentially the last question may lead to other pages so this may have to be tweaked but it may be done separately from the main steps generation or not)

Reply:
i'm looking at it from the point of view of the form wizard library: why would someone get me to check a condition and tell me it leads nowhere?

i'm not asking for a change, just raising a conceptual incongruity haha

Reply:
>last question may lead to other pages
that is probably best left up to the controller to decide in `getNextStep`